Examples/Python/src/Geometry.cpp

```diff
@@ -96,16 +121,200 @@ void addGeometry(Context& ctx) {
           return hook;
         }));
   }
+}
+
+void addExperimentaGeometry(Context& ctx) {
+  auto [m, mex] = ctx.get("main", "examples");
+
+  using namespace Acts::Experimental;
+
+  {  // Detector definition
+    py::class_<Detector, std::shared_ptr<Detector>>(m, "Detector");
+  }
+
+  {  // Detector volume definition
+    py::class_<DetectorVolume, std::shared_ptr<DetectorVolume>>(
+        m, "DetectorVolume");
+  }
```

Comment:
I think we dont have to scope those if it does not expose something
```suggestion
  // Detector definition
  py::class_<Detector, std::shared_ptr<Detector>>(m, "Detector");

  // Detector volume definition
  py::class_<DetectorVolume, std::shared_ptr<DetectorVolume>>(
      m, "DetectorVolume");
```

Reply:
Oki-doki.